### PR TITLE
Initial commit mobile repo

### PR DESCRIPTION
**Pull request by hallycsse**

### CODE REVIEW

**Review by BarnabePILLIAUDIN — CHANGES_REQUESTED** [review]
Can you please install and:
- nativewind
- eslint 
- prettier

And can you also remove default images and replace them by our logo

**Review by BarnabePILLIAUDIN — CHANGES_REQUESTED** [review]
@hallycsse 
Was not able to launch app
and add npm i step to the doc

File: mobile/app.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "ios": {
+      "supportsTablet": true
+    },
+    "android": {
+      "adaptiveIcon": {
+        "foregroundImage": "./assets/AirneisAdaptiveIcon.png",
+        "backgroundColor": "#ffffff"
+      }
+    },
+    "web": {
```
Comment (barnabepilliaudin) [inline mobile/app.json]: Is this line mandatory
If not can you please remove it

File: mobile/app.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      "favicon": "./assets/favicon.png"
```
Comment (barnabepilliaudin) [inline mobile/app.json]: Is this line mandatory
If not can you please remove it

File: mobile/app.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
```
Comment (barnabepilliaudin) [inline mobile/app.json:23]: Is this line mandatory
If not can you please remove it

**Review by BarnabePILLIAUDIN — COMMENTED** [review]
Removes the lines if they are not mandatory please

**Review by Magali-D — APPROVED** [review]

**Review by BarnabePILLIAUDIN — APPROVED** [review]
LGTM!

**Pull request merged**